I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 290, in script
    e "Line 1"
Exception: Image 'button' does not accept attributes 'left idle'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 290, in script
    e "Line 1"
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/ast.py", line 2572, in execute
    Say.execute(self)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/exports/sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/character.py", line 1451, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/character.py", line 1102, in do_display
    display_say(who,
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/character.py", line 799, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/core.py", line 2188, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/core.py", line 2705, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/screen.py", line 485, in visit_all
    self.child.visit_all(callback, seen=None)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 424, in visit_all
    for d in self.visit():
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/image.py", line 540, in visit
    self.find_target()
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 249, in _duplicate
    args.extraneous()
  File "/home/gaffy/Downloads/renpy-8.2.3-sdk/renpy/display/displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'button' does not accept attributes 'left idle'.

Linux-5.15.0-119-generic-x86_64-with-glibc2.35 x86_64
Ren'Py 8.3.0.24082114
TESTimony 1.0
Thu Aug 29 12:03:43 2024
